rpc/module.go

```diff
@@ -45,7 +45,10 @@ func (m *rpcModule) Create(runtime modules.RuntimeMgr) (modules.Module, error) {
 }
 
 func (u *rpcModule) Start() error {
-	go NewRPCServer(u.GetBus()).StartRPC(u.config.GetPort(), u.config.GetTimeout())
+	u.logger = logger.Global.CreateLoggerForModule(u.GetModuleName())
```

Comment:
Can you update the shared documentation in `shared/modules/doc/README.md` that new modules should do this?

Reply:
Doing so!